packages/gatsby/src/utils/__tests__/api-runner-node.js

```diff
@@ -160,8 +160,8 @@ it(`Ends activities if plugin didn't end them`, async () => {
   })
   await apiRunnerNode(`testAPIHook`)
 
-  expect(mockActivity.start).toBeCalledTimes(6)
+  expect(start).toBeCalledTimes(6)
```

Comment:
Because activity is being mutated, `mockActivity.start` is no longer mocked after passing though api-runner, but we have reference to original `start` so we can use that. Same for the `end` case